### PR TITLE
ticdc: fix alert doc

### PR DESCRIPTION
**Pull request by CharlesCheung96**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Keep the doc consistent with https://github.com/pingcap/tiflow/blob/master/metrics/alertmanager/ticdc.rules.yml. Note that alert rules diffes in diffrent release branches.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.5 (TiDB 8.5 versions)
- [ ] v8.4 (TiDB 8.4 versions)
- [ ] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ticdc/ticdc-alert-rules.md [hunk 1]
```diff
@@ -169,15 +141,15 @@ summary: 了解 TiCDC 集群监控报警规则以及处理方法。
 
     收集 TiCDC 监控和 TiKV 日志，定位原因。
 
-### `ticdc_sink_mysql_execution_error`
+### `ticdc_sink_execution_error`
```
Comment (hfxsd) [inline ticdc/ticdc-alert-rules.md:144]: 这条英文 PR 里漏掉了

**Review by hfxsd — APPROVED** [review]

File: ticdc/ticdc-alert-rules.md [hunk 2]
```diff
@@ -85,61 +71,47 @@ summary: 了解 TiCDC 集群监控报警规则以及处理方法。
...
 
 * 报警规则：
 
-    `changes(ticdc_processor_checkpoint_ts[1m]) < 1`
+    `(max_over_time(ticdc_owner_status[1m]) == 1 or max_over_time(ticdc_owner_status[1m]) == 6) > 0`
 
 * 规则描述：
-
-    TiCDC 某一个同步任务进度超过 1 分钟没有推进。
+
```
Comment (qiancai) [inline ticdc/ticdc-alert-rules.md:95]:
```suggestion

```

**Review by qiancai — APPROVED** [review]
Rest LGTM

**Pull request merged**